modules/ROOT/pages/release-notes.adoc

```diff
@@ -119,6 +119,18 @@ The property allowing to enable/disable xref:identity:rest-api-authorization.ado
 
 == Bug fixes
 
+=== Fixes in Bonita 2024.1-u1 (2024-05-30)
+
+==== Fixes in Bonita Runtime (including Bonita Applications)
+
+* RUNTIME-1844 - Update Tomcat to latest 9.0.87
+* RUNTIME-1833 - SanitizerFilter removes html tags from HTTP payload 
+* RUNTIME-1835 - SInvalidExpressionException generated when a parameter is used from within a groovy script
+* RUNTIME-1848 - ArchivedTasks filter on type is generating java.lang.ClassCastException
+* RUNTIME-1860 - Access Control panel is displayed in all subscription editions even when the feature is not active
+* RUNTIME-1864 - SCA cannot be started without BDM_ACCESS_CONTROL license key activated
+* CVE-63 - HTTP/2 CONTINUATION Flood vulnerability
```

Comment:
Do we usually show the fixed CVE?

Reply:
> Do we usually show the fixed CVE?

https://documentation.bonitasoft.com/bonita/2023.1/release-notes